package.json

```diff
@@ -173,7 +173,7 @@
     "prism-react-renderer": "^1.0.1",
     "query-string": "^6.14.0",
     "raw-loader": "^3.1.0",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
     "react-codesandboxer": "^3.1.5",
     "react-compare-image": "^2.0.3",
     "react-dom": "^17.0.2",
```

Comment:
react upgrades should also update `react-dom` to the same version

Reply:
@types packages should also be updated https://github.com/uber/baseweb/pull/5268/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L109-L110